fMRIVolume/scripts/DistortionCorrectionAndEPIToT1wReg_FLIRTBBRAndFreeSurferBBRbased.sh

```diff
@@ -79,6 +79,7 @@ Usage() {
   echo "                 don't do bias correction"
   echo ""
   echo "             --usejacobian=<\"true\" or \"false\">"
+  echo "             --preregister=<epi_reg (default) or flirt>"
```

Comment:
I mildly favor changing this to `--preregistertool` or `--preregistercmd`.  (`--preregister` by itself implies more of a TRUE/FALSE/YES/NO flag to me).
Also, while we are at it, please change the earlier 
`echo "             --method=<method used for readout distortion correction>"`
line to
`echo "             --method=<method used for susceptibility distortion correction>"`

Reply:
Done and done.